tools/webcam/README.md

```diff
@@ -1,37 +1,35 @@
-# Run openpilot with webcam on PC
+# Run openpilot with webcam on PC (TODO: test these steps on a fresh Ubuntu install to verify)
 
 What's needed:
-- Ubuntu 24.04
+- Ubuntu 24.04 ([WSL2 is not yet supported](https://github.com/commaai/openpilot/issues/34216))
 - GPU (recommended)
 - Two USB webcams, at least 720p and 78 degrees FOV (e.g. Logitech C920/C615)
 - [Car harness](https://comma.ai/shop/products/comma-car-harness) with black panda to connect to your car
 - [Panda paw](https://comma.ai/shop/products/panda-paw) or USB-A to USB-A cable to connect panda to your computer
 That's it!
 
 ## Setup openpilot
+- Follow [this readme](https://github.com/commaai/openpilot/tree/master/tools) to install and build the requirements
+- Install OpenCL Driver
 ```
-cd ~
-git clone https://github.com/commaai/openpilot.git
-```
-- Follow [this readme](https://github.com/commaai/openpilot/tree/master/tools) to install the requirements
-- Install [OpenCL Driver](https://registrationcenter-download.intel.com/akdlm/irc_nas/vcp/15532/l_opencl_p_18.1.0.015.tgz)
-
-## Build openpilot for webcam
-```
-cd ~/openpilot
-USE_WEBCAM=1 scons -j$(nproc)
+sudo apt install pocl-opencl-icd
 ```
 
 ## Connect the hardware
 - Connect the road facing camera first, then the driver facing camera
-- (default indexes are 1 and 2; can be modified in system/camerad/cameras/camera_webcam.cc)
 - Connect your computer to panda
 
 ## GO
 ```
-cd ~/openpilot/system/manager
-NOSENSOR=1 USE_WEBCAM=1 ./manager.py
+USE_WEBCAM=1 system/manager/manager.py
 ```
-- Start the car, then the UI should show the road webcam's view
-- Adjust and secure the webcams (you can run tools/webcam/front_mount_helper.py to help mount the driver camera)
+- Start the car, then the UI should show the road webcam's view (TODO: should it!? This needs fixing then)
+- Adjust and secure the webcams.
```

Comment:
The file referenced here can be restored with `git checkout c4f322ce2de191a07db992e649f73c128048bb74^ -- tools/webcam/front_mount_helper.py`
I couldn't get it to work for me on my janky WSL setup so I left it out.  Will test on native Ubuntu to verify

```
#!/usr/bin/env python
import numpy as np

# copied from openpilot.common.transformations/camera.py
eon_dcam_focal_length = 860.0  # pixels
webcam_focal_length = 908.0  # pixels

eon_dcam_intrinsics = np.array([
  [eon_dcam_focal_length,   0,   1152/2.],
  [  0,  eon_dcam_focal_length,  864/2.],
  [  0,    0,     1]])

webcam_intrinsics = np.array([
  [webcam_focal_length,   0.,   1280/2.],
  [  0.,  webcam_focal_length,  720/2.],
  [  0.,    0.,     1.]])

cam_id = 6

if __name__ == "__main__":
  import cv2  # pylint: disable=import-error

  trans_webcam_to_eon_front = np.dot(eon_dcam_intrinsics, np.linalg.inv(webcam_intrinsics))

  cap = cv2.VideoCapture(cam_id)
  cap.set(cv2.CAP_PROP_FRAME_WIDTH, 1280)
  cap.set(cv2.CAP_PROP_FRAME_HEIGHT, 720)

  while (True):
    print('hi')
    ret, img = cap.read()
    if ret:
      img = cv2.warpPerspective(img, trans_webcam_to_eon_front, (1152, 864), borderMode=cv2.BORDER_CONSTANT, borderValue=0)
      img = img[:, -864//2:, :]
      cv2.imshow('preview', img)
      cv2.waitKey(10)
```

Reply:
GREAT idea!  Will work on a follow-up PR with easy-to-follow user instructions